dair_pll/drake_utils.py

```diff
@@ -56,9 +56,8 @@
 CAM_FOV = np.pi/5
 VIDEO_PIXELS = [480, 640]
 FPS = 30
-SENSOR_POSE = RigidTransform(RollPitchYaw([-np.pi/2, 0, np.pi/2]), [1, 0, 0.2])
-EXP_NAME = file_utils.EXP_NAME
-VIDEO_FILENAME = file_utils.VIDEO_FILENAME
+SENSOR_POSE = RigidTransform(RollPitchYaw([-np.pi/2, 0, np.pi/2]), [2, 0, 0.2])
```

Comment:
This is an issue from my code earlier, not this PR, but the hard-coded camera pose deserves a `TODO` comment to dynamically set it based on the volume swept by the actual trajectory. I think this PR makes sense as a time to add it